### DISCUSSION TITLE
All functions in $refs.view

### DISCUSSION THREAD

**Discussion by WubbzyFromWuzzleburg** [Kitware/trame#298] [Q&A]
I'm working on a project which is similar to Paraview using Trame, VTK and Vue.js, recently I'm implementing functions that can set the view directions. I'm wondering how to find all the functions in "$refs.view"? I only know "$refs.view.resetCamera()" can reset the size and the location of the VTK image, I searched for reference but cannot find it.
Below is a small part of my code:
`from trame.widgets import vuetify`
`def tool_bar_icon():`
`# the button used to reset the camera`
`    with vuetify.VBtn(icon=True, click="$refs.view.resetCamera()"):`
`        vuetify.VIcon("mdi-arrow-expand-all")`

**Comment by jourdain** [comment]
I guess it depend on which view you are talking about. For client/server ones, it is usually preferred to perform the modification on the C++ implementation and ask to push the changes to the client.
